components/item/MediaPresentation.vue

```diff
@@ -17,7 +17,7 @@
       <iframe
         data-qa="media player"
         allowfullscreen="true"
-        :src="$path({ name: 'media', query: { id: europeanaIdentifier, mediaUrl: media.about } })"
+        :src="$path({ name: 'media', query: { id: europeanaIdentifier, mediaUrl: media.about, dashRequired: requiresDashJS } })"
```

Comment:
Remove this and replace with `mediaType: media.ebucoreHasMimeType`. Then make the media page responsible for performing the check against `requiresDashJS`.

Reply:
ok thanks, have updated